spec/lib/administrate/order_spec.rb

```diff
@@ -36,7 +36,7 @@
 
         ordered = order.apply(relation)
 
-        expect(relation).to have_received(:order).with(name: :asc)
+        expect(relation).to have_received(:order).with("name asc")
```

Comment:
Do you think there would be any value in testing for both sorts? (i.e.: string _and_ symbols)